encord/collection.py

```diff
@@ -39,12 +38,12 @@ def __init__(self, client: ApiClient, orm_collection: OrmCollection):
         self._collection_instance = orm_collection
 
     @property
-    def uuid(self) -> uuid.UUID:
+    def uuid(self) -> UUID:
```

Comment:
All other instances in this file use UUID from the package and can cause confusion.